native/python_package/python_driver/requestprocessor.py

```diff
@@ -30,6 +30,14 @@ class RequestCheckException(Exception):
     """
     pass
 
+class EmptyCodeException(Exception):
```

Comment:
It seems that this exception replaces the `RequestCheckException` one (I've not seen it used at least). If that's the case, remove the old one